src/components/Footer.jsx

```
@@ -1,286 +1,150 @@
+                    className="group relative inline-flex flex-col"
                 >
-                  <svg
-                    width="20"
-                    height="20"
-                    viewBox="0 0 24 24"
-                    fill="currentColor"
-                  >
-                    <path d="M12 0C8.74 0 8.333.015 7.053.072 5.775.132 4.905.333 4.14.63c-.789.306-1.459.717-2.126 1.384S.935 3.35.63 4.14C.333 4.905.131 5.775.072 7.053.012 8.333 0 8.74 0 12s.015 3.667.072 4.947c.061 1.278.263 2.148.567 2.913.306.789.717 1.459 1.384 2.126s1.337 1.079 2.126 1.384c.765.295 1.635.497 2.913.558C8.333 23.988 8.74 24 12 24s3.667-.015 4.947-.072c1.278-.061 2.148-.263 2.913-.567.789-.306 1.459-.717 2.126-1.384s1.079-1.337 1.384-2.126c.295-.765.497-1.635.558-2.913.012-1.278.072-1.687.072-4.947s-.015-3.667-.072-4.947c-.061-1.278-.263-2.148-.567-2.913-.306-.789-.717-1.459-1.384-2.126S20.65 1.079 19.86 0.772c-.765-.295-1.635-.497-2.913-.558C15.667.012 15.26 0 12 0zm0 2.16c3.2 0 3.583.012 4.85.07 1.17.055 1.834.249 2.227.4.597.23 1.05.545 1.432.928.382.383.698.834.928 1.432.15.393.345 1.057.4 2.227.058 1.267.07 1.65.07 4.85s-.012 3.583-.07 4.85c-.055 1.17-.249 1.834-.4 2.227-.23.597-.545 1.05-.928 1.432-.383.382-.834.698-1.432.928-.393.15-1.057.345-2.227.4-1.267.058-1.65.07-4.85.07s-3.583-.012-4.85-.07c-1.17-.055-1.834-.249-2.227-.4-.597-.23-1.05-.545-1.432-.928-.382-.383-.698-.834-.928-1.432-.15-.393-.345-1.057-.4-2.227-.058-1.267-.07-1.65-.07-4.85s.012-3.583.07-4.85c.055-1.17.249-1.834.4-2.227.23-.597.545-1.05.928-1.432.383-.382.834-.698 1.432-.928.393-.15 1.057-.345 2.227-.4 1.267-.058 1.65-.07 4.85-.07zm0 3.673c-3.4 0-6.16 2.76-6.16 6.16s2.76 6.16 6.16 6.16 6.16-2.76 6.16-6.16-2.76-6.16-6.16-6.16zm0 10.16c-2.208 0-4-1.792-4-4s1.792-4 4-4 4 1.792 4 4-1.792 4-4 4zm6.47-.988c-.689 0-1.25.56-1.25 1.25s.56 1.25 1.25 1.25 1.25-.56 1.25-1.25-.56-1.25-1.25-1.25z" />
-                  </svg>
+                    <span className="text-[#666] font-mono text-xs uppercase tracking-widest mb-2 group-hover:text-[#ccff00] transition-colors">
+                        // Initiate Communication
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)